namer/namer.cc

```diff
@@ -2020,6 +2020,20 @@ class TreeSymbolizer {
             }
         }
 
+        // Prevent a crash by validating the arguments before we do any symbol lookups or manipulations
+        if (send->hasPosArgs() || send->hasKwArgs() || send->block() != nullptr) {
+            if (send->numPosArgs() > 1) {
+                if (auto e = ctx.beginError(send->loc, core::errors::Namer::InvalidTypeDefinition)) {
+                    e.setHeader("Too many args in type definition");
```

Comment:
This error message could be a little better:

- ideally it mentions “type member” or “type template” because there are other ways to define types than just generic types
- Ideally the error header interpolates the name of the bad constant somewhere
- Ideally, there’s an autocorrect to delete the extra arguments, using e.replaceWith (to either replace the extra arguments with `""` or to replace all the arguments with the first argument (the latter handles the edge case of also deleting any comma between arg 1 and 2)